index.js

```diff
@@ -95,7 +95,9 @@ AFRAME.registerComponent('meshline', {
       );
     });
     
-    var widthFn = new Function ('p', 'return ' + this.data.lineWidthStyler);
+    var widthFn = this.data.lineWidthStyler && this.data.lineWidthStyler > 0
```

Comment:
Does this check work for "1 - p" as a lineWidthStyler function string ?

Reply:
Sorry, must have been very tired when I wrote that -- I meant to write `this.data.lineWidthStyler.length > 0`. I did not check whether `this.data.lineWidthStyler` is a string because the schema prescribes that anyway, but now that I am modifying this part of the code anyway, how about this one:

```
typeof this.data.lineWidthStyler === 'string' && this.data.lineWidthStyler.length > 0
```

Would that be okay? Let me know and I'll update the PR.

Reply:
looks good. could you test the build in your branch and update the PR, please ?

Reply:
`npm run build`, `npm run dist` and `npm run dev` seem to run fine for me on my machine. PR updated.